src/main/webapp/app/app.module.ts

```diff
@@ -97,18 +96,17 @@ import { ArtemisAdminModule } from 'app/admin/admin.module';
         ArtemisHomeModule,
         ArtemisAdminModule,
         ArtemisLegalModule,
+        ArtemisProgrammingSubmissionModule.forRoot(),
```

Comment:
Singleton services that are shared between a lazy loaded module and other modules, need to be initialized on root.